### ISSUE TITLE
Allow Projects to set an admin/invite-only expense policy independent of their parent Collective.

### ISSUE THREAD

**Issue by natehn** [opencollective/opencollective#5954]
## Who is your user?

A collective with multiple projects or activities, where only one of their Projects is getting spammed with expenses.

## What are they trying to achieve?

_When a Project is being spammed, I want to set expenses for that Project only to admin/invite only, so I can keep track of legit requests without having to sift through tons of bad ones_

## How are they currently doing this?
 

They currently do not know a way to get rid of the bad requests

## How well understood is this problem?

 - [x] Surfacing: if you feel this is a seed of an idea that needs to be researched.
 - [ ] Understanding: if you've done some research and feel it is already well understood and ready for shaping.
 - [ ] Shaping:  if you feel a solution is clearly defined and can be made into a pitch ([read more about shaping](https://basecamp.com/shapeup/1.1-chapter-02))
 - [ ] Pitching: if you feel that it is ready to be presented for prioritisation ([read more about pitching](https://basecamp.com/shapeup/1.5-chapter-06))

**Comment by missjuliaho** [comment]
I am having a similar issue -- my collective settings are set to only allow team members to submit expenses but it does not apply to the project within my collective so I'm being spammed through the Project. Would like to either see collective settings automatically apply to all projects or allow projects to have their own settings / admins: https://github.com/opencollective/opencollective/issues/4611

**Comment by CiErra-hub** [comment]
Our collective has also had an issue of being spammed. It would be great if there were individual ways to control per project.

**Comment by SudharakaP** [comment]
I am in favor of simply applying all the policies from parent collective to the child collective. Let me know if there's any alternative suggestions. 

cc: @Betree

**Comment by Betree** [comment]
> I am in favor of simply applying all the policies from parent collective to the child collective. Let me know if there's any alternative suggestions. 
> 
> cc: @Betree 

cc @kewitz it sounds like this is what we're doing already with the latest changes to the policies?

**Comment by kewitz** [comment]
Yes, we recently implemented the policy inheritance between parent and child collective.
@SudharakaP could you please try to reproduce this and, if this is an acceptable solution, mark it as completed?

**Comment by znarf** [comment]
@kewitz Was still being reported 2 days ago? https://opencollective.slack.com/archives/C6JTTA4SK/p1677015014369879

**Comment by natehn** [comment]
This whole issue is about the Project _not_ inheriting!

**Comment by natehn** [comment]
Can you clarify what has changed here? The desire is for Projects to be able to set a different policy on this from their parent Collectives.

**Comment by kewitz** [comment]
@natehn we need a bit more information about this issue.

It seems we're working on allowing an independent expense policy from the parent collective, but the user comments in this issue suggest that simply inheriting the setting would solve the issue.

Considering that inheriting settings is the default behavior, technically speaking this is not only the simplest solution but also the right one because it is consistent with the expected behavior.

What exactly is the case for allowing the user to customize the event/project setting? i.e. in wich condition, the user wants to block users from submitting expenses for a project while allowing them to submit to the parent collective (and vice-versa)?

**Comment by natehn** [comment]
@kewitz Yes, this is exactly it! There are so many potential uses of Projects and Collectives, that inheriting the setting will only lead to more requests for features. I think the default being that it inherits is good, but that a Project should be able to restrict submitters even if its parent Collective does not, and vice versa.

**Comment by natehn** [comment]
This desire is actually in the title of the issue: Allow Projects to set an admin/invite-only expense policy **independent of their parent Collective**.
